src/Text/Ginger/Run/Builtins.hs

```diff
@@ -188,6 +188,9 @@ looseEquals a b
     | isNull a || isNull b = asBoolean a == asBoolean b
     | otherwise = asText a == asText b
 
+gfnIsLt :: Monad m => Function (Run p m h)
+gfnIsLt = gfnLess
+
```

Comment:
This function is unnecessary; you can just map context variables to `gfnLess` directly.